examples/java/src/test/java/org/apache/beam/examples/DebuggingWordCountTest.java

```diff
@@ -35,6 +35,13 @@
 public class DebuggingWordCountTest {
   @Rule public TemporaryFolder tmpFolder = new TemporaryFolder();
 
+  private String getFilePath(String filePath) {
+      if (filePath.contains(":")) {
+          return filePath.replace("\\", "/").split(":")[1];
```

Comment:
It sounds good to me. You can also use NIO `Paths.get()` that automatically does the conversion.